x/stakeibc/migrations/v3/convert.go

```diff
@@ -0,0 +1,86 @@
+package v3
+
+import (
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	oldstakeibctypes "github.com/Stride-Labs/stride/v13/x/stakeibc/migrations/v3/types"
+	newstakeibctypes "github.com/Stride-Labs/stride/v13/x/stakeibc/types"
+)
+
+var (
+	// The threshold, denominated in percentage of TVL, of when a slash query should
+	// be submitted (1 => 1%)
+	ValidatorSlashQueryThreshold uint64 = 1
+	// The exchange rate here does not matter since it will be updated after the slash query
+	// Setting it to this value makes it easier to verify that we've submitted the query
+	DefaultExchangeRate = sdk.MustNewDecFromStr("1.000000000000001")
```

Comment:
Won't this rate be used in the RR if we hit an epoch before a first slash query to this validator?

Reply:
(Separate) nit: I think exchRate g.t. 1 is impossible so might make sense to make the default (even if unused) something like 0.999 instead.

Reply:
It's only used in the redemption rate when summing up LSMTokenDeposits, but we should run the query on all validators before LSM is even live on the hub

Reply:
also, agreed, 0.9999 probably makes more sense

Reply:
Cool this makes sense